src/xrdsvc/SsiService.h

```diff
@@ -57,14 +55,13 @@ namespace lsst::qserv::xrdsvc {
 /// worker services
 class SsiService : public XrdSsiService {
 public:
-    /** Build a SsiService object
-     *
+    /**
+     * Build a SsiService object
```

Comment:
I think the old version is legal and makes that first statement equivalent to "@brief  Build a SsiService object"